packages/default/scss/common/utils/_flex.scss

```diff
@@ -0,0 +1,46 @@
+@include exports( "common/utils/flex" ) {
```

Comment:
Which component uses these utility classes?

Reply:
For both questions the answer is the upcoming chat and card components

Reply:
Sounds good. In that case I suggest adding these classes along with the PR for the chat and card components, along with the example usage, and using a `feat:` commit to notify customers of the new theme features.

Reply:
Done